service/controller/v10/resource/containerurl/create.go

```diff
@@ -24,7 +26,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	_, err = storageAccountsClient.GetProperties(ctx, groupName, storageAccountName)
+	_, err = storageAccountsClient.GetProperties(ctx, groupName, storageAccountName, storage.AccountExpandGeoReplicationStats)
```

Comment:
`AccountExpandGeoReplicationStats` is the only option at the moment but anyway we need to add this as parameters.